jupyterhub_moss/form/option_form.css

```diff
@@ -93,4 +105,24 @@
   .form-container input[type="checkbox"] {
     grid-column: 2;
   }
+
+  .form-offset-row {
+    width: 100%;
+  }
+}
+
+.environment-div {
+  padding: 0.15rem;
+  display: flex;
+  align-items: flex-start;
+  gap: 0.3rem;
```

Comment:
This is not going to be supported on many browsers. You can use padding/margin on the children instead.